clarifai/client/input.py

```diff
@@ -908,10 +908,13 @@ def _retry_uploads(self, failed_inputs: List[Input]) -> None:
     """Retry failed uploads.
 
     Args:
-        failed_inputs (List[Input]): failed input prots
+        failed_inputs (List[Input]): failed input protos
+        max_retries (int): max retries for failed uploads
     """
-    if failed_inputs:
-      self._upload_batch(failed_inputs)
+    max_retries = 2
```

Comment:
Can we use the same _MAX_RETRIES_ constant we defined in `constants/dataset.py`

Reply:
@sanjaychelliah ,
 Yeah I was hesitant that whether for Input, should we create new constant file or to use the dataset constant file itself ? does it create any circular import issue ? just asking !

Reply:
We can use the same one in dataset constant. Test it once, I think it'll be fine..